functionals.py

```diff
@@ -1,12 +1,15 @@
+import os
+import sys
 import json
+import errno
```

Comment:
We stopped using the `force_symlink` function so this import is no longer needed.